modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -90,20 +88,18 @@ def apply(self, func, *args, **kwargs):
         -----
         The keyword arguments are sent as a dictionary.
         """
-        client = default_client()
         call_queue = self.call_queue + [[func, args, kwargs]]
         if len(call_queue) > 1:
-            future = client.submit(
-                apply_list_of_funcs, call_queue, self.future, pure=False
+            futures = DaskWrapper.deploy(
+                apply_list_of_funcs, 2, call_queue, self.future, pure=False
```

Comment:
We shouldn't use magic constants. Let's add a variable in line above like `num_returns` and put a comment nearby why the variable is equal to 2.

Reply:
The same problem is observed when using `deploy` function in other places.

The only way I can see to improve readability is to change the interface so that this option can be used in this way: `num_returns=2`.

In your case, there will be many places where these variables (`num_returns`) will be defined before `deploy` call with the same comments like "the number 2 is used here, because such and such a function returns two arguments"

Reply:
The use of num_returns works for me.

Reply:
added `num_returns` variable